back/sssscs/src/main/java/com/ib/common/EntityNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.ib.common;
+
+public class EntityNotFoundException extends EntityException {
+
+    public EntityNotFoundException(Class entityType, Long id) {
```

Comment:
Eclipse is complaining that this should be parameterized, like ```Class<?>```. Is there a reason why it's just `Class` here (and in all the other files)?

Reply:
Missed it since IntelliJ only shows warnings when you're in the specific file, not project-wide like Eclipse
https://github.com/magley/sssscs/pull/2/commits/0c2faad0e93d9f57f4fe555dcd8e400fcb1ac2f7